src/Http/Response.php

```diff
@@ -0,0 +1,2432 @@
+<?php
```

Comment:
Looks like that move didnt work, it is a delete + add.
Wouldn't that kill the history on the file content?